selda/src/Database/Selda/Migrations.hs

```diff
@@ -137,10 +137,10 @@ migrateInternal t1 t2 upg = withBackend $ \b -> do
     void . liftIO $ runStmt b renameQuery []
     createTableIndexes Fail t2
   where
-    t2' = t2 {tableName = mkTableName newName} `asTypeOf` t2
-    newName = mconcat ["__selda_migration_", rawTableName (tableName t2)]
+    t2' = t2 {tableName = newName} `asTypeOf` t2
+    newName = mkTableName $ mconcat ["__selda_migration_", rawTableName (tableName t2)]
     renameQuery = mconcat
-      [ "ALTER TABLE ", newName
+      [ "ALTER TABLE ", fromTableName newName
```

Comment:
@YPares just curious, is there any reason why taking this through `mk`&`from`-`TableName` is more viable? (sounds same to me so probably OK.)